behaviortree_ros2/include/behaviortree_ros2/bt_topic_sub_node.hpp

```diff
@@ -75,6 +76,7 @@ class RosTopicSubNode : public BT::ConditionNode
     rclcpp::CallbackGroup::SharedPtr callback_group;
     rclcpp::executors::SingleThreadedExecutor callback_group_executor;
     boost::signals2::signal<void (const std::shared_ptr<TopicT>)> broadcaster;
+    std::shared_ptr<TopicT> last_msg = nullptr;
```

Comment:
Already null by default.
```suggestion
    std::shared_ptr<TopicT> last_msg;
```